### ISSUE TITLE
Add image to data labels for Line graph.

### ISSUE THREAD

**Issue by vishalpathak** [core-plot/core-plot#71]
I want add an image to the bag-round of data label in line graph. And for plotting data labels I'm using following method 
-(CPTLayer *) dataLabelForPlot:(CPTPlot *)plot recordIndex:(NSUInteger)index
My values are getting plotted but I am unable to apply image. I want to show my graph like in this attached image.
![linegraph](https://f.cloud.github.com/assets/6926935/2394992/9c97c236-a9c1-11e3-8a78-20d0d22cd30d.png)
I want add callout shape image to my plotted data label.

**Comment by dddppp** [comment]
Have you found a way to do this? Also, how did you create the green area in the graph/plot? And how did you define the green thick plot style?

**Comment by vishalpathak** [comment]
yes I found the way and added the green area on data labels but what is
your requirement same as mine?
